docs/wiki/basic-tutorials/interactions/commands.md

```diff
@@ -252,6 +252,7 @@ api.batchUpdateSlashCommandPermissions(server, Arrays.asList(builder1, builder2)
 - Your app cannot have two server commands within the same name on the same guild
 - Your app can have a global and guild command with the same name
 - Multiple apps can have commands with the same names
+- Names cannot contain empty spaces
```

Comment:
What is an "empty" space?
Especially compared to a "non-empty" space.

Reply:
Poor choice of words on my part. I meant space " " symbol. 
I havent tested it with special characters, but if they forbid " ", i'd expect that it only allows a-Z and 0-9